qp_qiime2/qiime2.py

```diff
@@ -713,3 +713,71 @@ def emperor(qclient, job_id, parameters, out_dir):
     ainfo = [ArtifactInfo('q2_visualization', 'q2_visualization',
                           [(emperor_qzv, 'qzv')])]
     return True, ainfo, ""
+
+
+def beta_group_significance(qclient, job_id, parameters, out_dir):
+    """generate beta correlation calculations
```

Comment:
ANOSIM and PERMANOVA are not correlation calculations this should be `Generate beta group significance calculations` or something along those lines.